### PR TITLE
Update sdk-platform and v4-develop in urls

### PR DESCRIPTION
**Pull request by piiq**

### CODE REVIEW

File: website/pypi.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The command below provides access to the core functionalities behind the [OpenBB
 pip install openbb-terminal
 ```
 
-If you wish to utilize our **Portfolio Optimization** or **Machine Learning / Artificial Intelligence** toolkits, please see instructions [here](https://docs.openbb.co/sdk/installation).
+If you wish to utilize our **Portfolio Optimization** or **Machine Learning / Artificial Intelligence** toolkits, please see instructions [here](https://docs.openbb.co/platform/installation).
```
Comment (hjoaquim) [inline website/pypi.md:25]: Those toolkits are not yet supported on the Platform, should we still point this section to the platform installation procedure?

Comment (piiq) [reply website/pypi.md:25]: good catch. maybe just pointing to the docs root?

**Comment by piiq** [comment]
Thanks for proof-reading this @hjoaquim. I've reverted the changes that were wrongfully introduced for v3 documentation and changed the base branch of this PR to be the docs PR. This PR depends on the docs PR, so if it's merged the links won't be broken.

File: website/pypi.md (hunk 1 above)
Comment (piiq) [reply website/pypi.md]: reverted

File: website/pypi.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Access our fully fledged financial SDK with a single line of python code.
 from openbb_terminal.sdk import openbb
 ```
 
-Everything you need to use the OpenBB SDK can be found on our [Official Documentation](https://docs.openbb.co/sdk).
+Everything you need to use the OpenBB SDK can be found on our [Official Documentation](https://docs.openbb.co/platform).
```
[the remark this answers is not in the document]
Comment (piiq) [reply website/pypi.md]: reverted

**Pull request merged**